src/query/models/matcher.go

```diff
@@ -38,6 +38,10 @@ func (m MatchType) String() string {
 		return "=~"
 	case MatchNotRegexp:
 		return "!~"
+	case MatchField:
+		return "-"
```

Comment:
For my understanding, how did you come up with these mappings?

Reply:
Made 'em up :p

The only thing this affects is the `String()` method on MatchTypes